datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/dataplatforminstance/DataPlatformInstanceType.java

```diff
@@ -24,7 +24,15 @@
 
 public class DataPlatformInstanceType implements com.linkedin.datahub.graphql.types.EntityType<DataPlatformInstance, String> {
 
-    static final Set<String> ASPECTS_TO_FETCH = ImmutableSet.of();
+    static final Set<String> ASPECTS_TO_FETCH = ImmutableSet.of(
+        Constants.DATA_PLATFORM_INSTANCE_KEY_ASPECT_NAME,
+        Constants.DATASET_PROPERTIES_ASPECT_NAME,
+        Constants.DEPRECATION_ASPECT_NAME,
+        Constants.OWNERSHIP_ASPECT_NAME,
+        Constants.INSTITUTIONAL_MEMORY_ASPECT_NAME,
+        Constants.GLOBAL_TAGS_ASPECT_NAME,
```

Comment:
what do you think about adding `GLOSSARY_TERMS_ASPECT_NAME` and `DOMAINS_ASPECT_NAME` if the goal is to get this to have an entity profile page? we'd have to update entity-registry.yml to include those aspects for this entity as well

Reply:
I mentioned that in the previous PR, the one enabling the ingestion for all Data Platform Instance aspects https://github.com/datahub-project/datahub/pull/5728

> I have explicitely skipped GlossaryTerm and Domain because those look more related to data than infrastructure

WDYT? I would skip these until we have clear use cases for that. Adding them after this will be straightforward.

Reply:
that is totally good with me! it makes sense and is definitely something we can come back to later.